### DISCUSSION TITLE
Include extra files in build

### DISCUSSION THREAD

**Discussion by ConorHK** [BirdeeHub/nixCats-nvim#51] [Q&A] [answered]
How would I include extra files in the build output? I want to include a pyproject.toml for fallback as a default set of settings for the ruff lsp. I want that file to exist in a subdirectory of my vim config directory - so I need to somehow include it in the build output so it can be sourced if theres no pyproject.toml in the python directory. Any guidance?

Also nixCats is really cool, thank you!

**Comment by BirdeeHub** [comment, marked as answer]
Just put it somewhere in your nvim directory (the one pointed to by luaPath)

The whole thing gets [`cp -r *`](https://github.com/BirdeeHub/nixCats-nvim/blob/5ac064e2f786790e6a6e22d7d9bdfc56b75ad029/nix/builder/default.nix#L127-L134) into the store.

You may then refer to it in several ways but [`require('nixCats').configDir`](https://github.com/BirdeeHub/nixCats-nvim/blob/5ac064e2f786790e6a6e22d7d9bdfc56b75ad029/nix/builder/nixCats.lua#L6) is a good one
